src/core/contractInterpreters/SushiswapRouter_0xd9e1.json

```diff
@@ -0,0 +1,491 @@
+{
+    "contractAddress": "0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F",
+    "methods": {
+        "0xe8e33700": "addLiquidity(address,address,uint256,uint256,uint256,uint256,address,uint256)",
+        "0xf305d719": "addLiquidityETH(address,uint256,uint256,uint256,address,uint256)",
+        "0xbaa2abde": "removeLiquidity(address,address,uint256,uint256,uint256,address,uint256)",
+        "0x02751cec": "removeLiquidityETH(address,uint256,uint256,uint256,address,uint256)",
+        "0xaf2979eb": "removeLiquidityETHSupportingFeeOnTransferTokens(address,uint256,uint256,uint256,address,uint256)",
+        "0xded9382a": "removeLiquidityETHWithPermit(address,uint256,uint256,uint256,address,uint256,bool,uint8,bytes32,bytes32)",
+        "0x5b0d5984": "removeLiquidityETHWithPermitSupportingFeeOnTransferTokens(address,uint256,uint256,uint256,address,uint256,bool,uint8,bytes32,bytes32)",
+        "0x2195995c": "removeLiquidityWithPermit(address,address,uint256,uint256,uint256,address,uint256,bool,uint8,bytes32,bytes32)",
+        "0xfb3bdb41": "swapETHForExactTokens(uint256,address[],address,uint256)",
+        "0x7ff36ab5": "swapExactETHForTokens(uint256,address[],address,uint256)",
+        "0xb6f9de95": "swapExactETHForTokensSupportingFeeOnTransferTokens(uint256,address[],address,uint256)",
+        "0x18cbafe5": "swapExactTokensForETH(uint256,uint256,address[],address,uint256)",
+        "0x791ac947": "swapExactTokensForETHSupportingFeeOnTransferTokens(uint256,uint256,address[],address,uint256)",
+        "0x38ed1739": "swapExactTokensForTokens(uint256,uint256,address[],address,uint256)",
+        "0x5c11d795": "swapExactTokensForTokensSupportingFeeOnTransferTokens(uint256,uint256,address[],address,uint256)",
+        "0x4a25d94a": "swapTokensForExactETH(uint256,uint256,address[],address,uint256)",
+        "0x8803dbee": "swapTokensForExactTokens(uint256,uint256,address[],address,uint256)"
+    },
+    "contractOfficialName": "UniswapV2Router02",
```

Comment:
Does Sushi have an equivalent to this?

Reply:
Sushi's official contract name is the same thing. Like the contract Official name and all the individual contracts have the same methodIDs, but the router contract address is different